crates/nixci/src/command/run.rs

```diff
@@ -9,13 +9,21 @@ use nix_rs::{
     flake::{system::System, url::FlakeUrl},
     info::NixInfo,
 };
+use std::path::PathBuf;
 
 use crate::{
     config::core::Config,
     flake_ref::FlakeRef,
-    nix::system_list::{SystemsList, SystemsListFlakeRef},
+    nix::{
+        ssh,
+        system_list::{SystemsList, SystemsListFlakeRef},
+    },
 };
 
+/// We expect this environment to be set in Nix build and shell.
+pub const OMNIX_SOURCE: &str = env!("OMNIX_SOURCE");
```

Comment:
Cannot this be made of type `PathBuf` (to obviate the conversion below where it gets used) here?

Reply:
it's not possible to directly make it a const PathBuf because  `env! macro` returns a &'static str, which is a string slice. Also creating a `PathBuf` requires heap allocation, which can't be done at compile-time in a const context.